kahuna/public/js/search/query.js

```diff
@@ -44,6 +44,13 @@ query.controller('SearchQueryCtrl',
         {label: 'Past week',     value: pastWeek}
     ];
 
+    ctrl.orderByOptions = [
+        {label: 'Upload Time (desc)'},
+        {label: 'Upload Time (asc)', value: 'uploadTime'},
+        {label: 'Date Taken (desc)', value: '-dateTaken'},
+        {label: 'Date Taken (asc)',  value: 'dateTaken'}
```

Comment:
Something very useful that has come up is ordering by dateModified - would this be easy to wedge in?